test/master-playlist-controller.test.js

```diff
@@ -5534,53 +5534,6 @@ QUnit.module('MasterPlaylistController experimentalBufferBasedABR', {
   }
 });
 
-QUnit.test('Determines if playlist should be aborted on earlyabort', function(assert) {
```

Comment:
Should this test be left in just when `experimentalBufferBasedABR` is false?

Reply:
Maybe, though, this test was added as part of https://github.com/videojs/http-streaming/pull/886.

Reply:
Actually, looks like we do have a similar test that already https://github.com/videojs/http-streaming/blob/ca7655e0f8ef12f708367e0bedbde3d38e900659/test/master-playlist-controller.test.js#L1847-L1887. This test is a variation of that that accounts for `shouldSwitchToMedia` variations with `experimentalBufferBasedABR`